src/models/PolicyList.ts

```diff
@@ -86,6 +86,10 @@ class PolicyList extends EventEmitter {
     // Events that we have already informed the batcher about, that we haven't loaded from the room state yet.
     private batchedEvents = new Set<string /* event id */>();
 
+    /** MSC3784 support. Please note that policy lists predate room types. So there will be lists in the wild without this type. */
+    public static readonly ROOM_TYPE = "support.feline.policy.lists.msc.v1";
```

Comment:
`support.feline.policy.lists.msc.v1`? I assume that this was created for this occasion? We typically use `org.matrix.mjolnir.XXX` for Mjölnir-specific prefixes, no?

Reply:
This isn't my MSC, so we don't have a choice in the matter

Reply:
It's the unstable room type for MSC3784.